packages/connect-widget/src/shared/util.ts

```diff
@@ -17,6 +17,15 @@ export function redirectToMain(navigate: NavigateFunction, searchParams: URLSear
   }
 }
 
+// redirects to the custom redirect url
+export function redirectToCustomUrl(url: string) {
+  try {
+    window.location.href = url;
+  } catch (err) {
+    capture.error(err, { extra: { context: `redirectToCustomUrl` } });
```

Comment:
Add the `url` to `extra` as well, please, better to have as much info as possible in these